tests/RegressionTests_hera.intel.log

```diff
@@ -1,3 +1,27 @@
+<<<<<<< HEAD
```

Comment:
Please revert this log to 'develop' version. It will create conflict when we merge from develop right before we start running final set of regression tests.